### DISCUSSION TITLE
Loco Range

### DISCUSSION THREAD

**Discussion by esnelling** [bitcraze/discussions#1299] [Q&A]
I'd really like to setup a system with Loco for indoor light shows but the show area seems super small even with Loco. Can the anchors be expanded greater than the small containment area shown in the Crazyflie documentation? If so, what it the realistic 3D area?  If you look at the Litebee system, their positioning systems only uses 4 anchors and seems to have quite a large coverage area. Thanks!

**Comment by knmcguire** [comment]
Yes, it is possible to make a larger area for sure. We fly in a 10x10x3 flight area and we managed to get a 25 by 25 by 9 meters with more anchors for an experimental indoor show. I'm sure that less anchors and perhaps more distance can be made, but the Crazyflie is not made for drone shows in mind but is an development and educational platform. So we hope that researchers will contribute any improvements to the system back. 

Please checkout this blogpost about the big loco show but disclaimer, it was experimental and required quite some tuning (this will not run as is out of the box) : https://www.bitcraze.io/2022/12/the-big-loco-show/

**Comment by esnelling** [comment]
Thank you very much Kimberly. Yes, that was a very interesting project. I
am very hopeful for the Crazyflie platform as it seems to have some good
support with Skybrush. The pain points right now are:

   1. Limited flight time
   2. Very noisy for indoors for artistic projects
   3. Too many anchor points required for UWB and long setup time. I'd
   really love to know how other OEMs accomplish such good position reference
   over a wide area with only 4 UWB anchors at ~2m

I think you know all of these already and I totally understand light shows
are not the designed use-case for Crazyflie.

Many thanks for your reply. Wonderful product you have developed.

**Reply by knmcguire** [reply]
About the noise and limited flight time there is not much we can do about that. And what litebee is exactly doing is difficult to say as their firmware is not opensource. What I can 'guess' is that in addition of UWB the litebee is fusing info with a height range sensor and optical flow.
